lms/views/basic_lti_launch.py

```diff
@@ -166,6 +166,50 @@ def db_configured_basic_lti_launch(self):
         ).document_url
         return self.basic_lti_launch(document_url)
 
+    @view_config(db_configured=True, is_canvas=True)
```

Comment:
Did the split using a predicate here. Other alternatives work as well but I think predicates work nicely to route based on url params and similar.